pkg/handlers/generic/lifecycle/utils/utils_suite_test.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2023 D2iQ, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestAMIPatch(t *testing.T) {
```

Comment:
Did you forget to change the name?

Reply:
Are you suggesting I copy-pasted this and forgot to change the name? How dare you! :joy:

Reply:
Thanks for catching this. I fixed the function, and moved the integration tests to their own file to decrease confusion with the unit tests.